testing/cli-e2etest/helpers/table_unmarshal.go

```diff
@@ -0,0 +1,67 @@
+package helpers
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+type tableColumnMetadata struct {
+	Name string
+	Len  int
+}
+
+func UnmarshalTable(t *testing.T, data string) []map[string]string {
```

Comment:
With this method, we can validate the table output in our tests. I'm planning to do that in future PRs.